contracts/BaseGasSponsor.sol

```diff
@@ -9,15 +9,33 @@ import "./interfaces/IRelayHub.sol";
 /**
  * Abstract base class to be inherited by a concrete Gas Sponsor
  * A subclass must implement:
- *  - acceptRelayCall
+ *  - acceptRelayedCall
  *  - preRelayedCall
  *  - postRelayedCall
  */
 contract BaseGasSponsor is IGasSponsor, Ownable {
 
+    // Gas stipends for acceptRelayedCall, preRelayedCall and postRelayedCall
+    uint256 constant private ACCEPT_RELAYED_CALL_MAX_GAS = 50000;
```

Comment:
Why are they private?

Reply:
Well, there is a getter right there, why wouldn't you use it?